### ISSUE TITLE
Отчеты в FastReport4

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#1622]
Originally reported on Google Code with ID 1622

```
Если в отчете построенном в FastReport4 нажать кнопку свернуть отчет 
закрывается полностью. А иногда хочется сравнить 2-а отчета не распечатывая 
их. 
```

Reported by `www.sasha` on 2009-09-23 14:44:07

**Comment by gsbelarus** [comment]
Reported by `Alexander.GoldenSoft` on 2009-09-23 17:01:38
- **Status changed**: `Duplicate`
- **Merged into**: #1427

**Comment by gsbelarus** [comment]
Reported by `Alexander.GoldenSoft` on 2009-09-24 22:04:21
- **Status changed**: `Fixed`
- **Merged into**: #-1427
